test/jdk/jdk/jfr/event/oldobject/TestObjectDescription.java

```diff
@@ -41,7 +41,9 @@
  * @test
  * @key jfr
  * @requires vm.hasJFR
- * @requires vm.gc != "Z" & vm.gc != "Shenandoah"
```

Comment:
Why did you remove the exclusion of ZGC here and left Shenandoah? Isn't it better to leave the ZGC exclusion here?

Reply:
The ZGC is now generational, and the OldObjectSampler should work.

Reply:
I don't know what the generational part has to do with it. The Old part in the name doesn't refer the young and old generation, just that an object is lingering in the system, which it also did for single gen ZGC.

Reply:
Thanks, I copied your comments to the bug. It might be needed to run specific tests for Parallel/Serial GC like for ZGC so this will be just G1 only.